backend/index.ts

```diff
@@ -1,12 +1,82 @@
 const express = require('express');
+const mongoose = require('mongoose');
+const bodyParser = require('body-parser');
+
 const app = express();
+app.use(bodyParser.json());
+
+// Connect to MongoDB
+mongoose.connect(
+  'mongodb://localhost/mydatabase',
+  { useNewUrlParser: true }
+);
+
+const itemSchema = new mongoose.Schema({
+  name: String,
+  description: String,
+});
 
-const port = process.env.PORT || 3001;
+const Item = mongoose.model('peoples', itemSchema);
 
-app.get('/', (req, res) => {
-  res.send('Dummy API');
+app.post('/add-item', async (req, res) => {
```

Comment:
Follow the same notation 
Replace /add-item with -> /addBlog
/update-item with -> /updateBlog
Similarly /deleteBlog and /getBlog

Reply:
Got it.